stix_shifter_modules/qradar/stix_translation/json/aql_flows_fields.json

```diff
@@ -71,6 +71,7 @@
     "\"tls ja3s hash\" as tlsja3shash",
     "\"suspect content descriptions\" as suspectcontentdescriptions",
     "flowid",
-    "\"content type\" as contenttype"
+    "\"content type\" as contenttype",
+    "hasOffense"
```

Comment:
Would this be different from `hasoffense` which is also in this file?

Reply:
not at all, missed the older one. The mapping for this field is missing in to_stix and from_stix files, so the only place to remove is here.